packages/fxa-settings/src/components/Settings/PageRecoveryPhoneRemove/index.tsx

```diff
@@ -0,0 +1,102 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { Link, RouteComponentProps, useNavigate } from '@reach/router';
+import VerifiedSessionGuard from '../VerifiedSessionGuard';
+import { SETTINGS_PATH } from '../../../constants';
+import CardHeader from '../../CardHeader';
+import AppLayout from '../../AppLayout';
+import Banner from '../../Banner';
+import LinkExternal from 'fxa-react/components/LinkExternal';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { useAccount } from '../../../models';
+
+// TODO, update this link with #section-heading once the SUMO article is updated (FXA-10918)
+const sumoTwoStepLink = (
+  <LinkExternal
+    className="link-blue"
+    href="https://support.mozilla.org/en-US/kb/secure-firefox-account-two-step-authentication"
+  >
+    Compare recovery methods
+  </LinkExternal>
+);
+
+const PageRecoveryPhoneRemove = (props: RouteComponentProps) => {
+  const navigate = useNavigate();
+  const account = useAccount();
+  // TODO, actually get this number back and format it
+  const formattedFullPhoneNumber = '+1 (555) 555-5555';
+
+  const goHome = () => navigate(SETTINGS_PATH + '#security', { replace: true });
+
+  return (
+    <AppLayout>
+      <VerifiedSessionGuard onDismiss={goHome} onError={goHome} />
+      <CardHeader
+        headingText="Remove recovery phone number"
+        headingTextFtlId="recovery-phone-remove-header"
+      />
+      <FtlMsg
+        id="settings-recovery-phone-remove-info"
+        vars={{ formattedFullPhoneNumber }}
```

Comment:
Was wondering if we need to include the `elems` attribute for the <strong> tag, but I see other instances like this in our codebase so maybe not

Reply:
Though... when I look at storybook, it's not showing as bold so maybe it is needed?

![image](https://github.com/user-attachments/assets/d5ea052b-eb9a-4e24-9896-b39b5e242913)

Reply:
Updated.